### DISCUSSION TITLE
Automatically apply dos2unix to startup-config and partial

### DISCUSSION THREAD

**Discussion by tiago-amado** [srl-labs/containerlab#1983] [Ideas]
Clab returns an error when we upload config files from windows to unix. 
To avoid that either we change the EOL format from windows to unix before upload or we execute dos2unix on the server after upload the files.
It would be nice having CLAB doing the dos2unix  for the config files instead of return the error.

````yml
ERRO[0168] failed to run postdeploy task for node edge-l-104: failed to apply config; error: errTimeoutError: channel timeout sending input to device
ERRO[0168] failed to run postdeploy task for node edge-s-112: failed to apply config; error: errTimeoutError: channel timeout sending input to device
````

**Comment by steiler** [comment]
You always have these kind of issues when transferring text files between unix and windows. I hence do not consider it clabs task to manage users file encodings.
You should maybe use a more advanced editor then notepad (e.g.  [notepad++](https://notepad-plus-plus.org/downloads/) or many others) and already on windows save and store them in unix format.

**Reply by tiago-amado** [reply]
Agree that its not CLAB task to manage this, but would be very nice having CLAB doing it! (if the required effort for this is low) 
I always use notepad++ and already change the default settings to use unix, but I saw many colleagues spent hours troubleshooting this issue, and  I also faced this in the past.
